TCMSApp/src/client/app/tests/tests.html

```diff
@@ -19,17 +19,19 @@
       <div class="col-xs-3">
         <div class="panel panel-info">
           <div class="panel-heading">
-            <h2>Suites <button type="button" class="btn btn-primary btn-xs" ng-click="vmTests.open()">NEW</button></h2>
+            <h3>Suites <button type="button" class="btn btn-primary btn-xs" ng-click="vmTests.open()">NEW</button></h3>
+          </div>
+          <div class="table-responsive">
+            <table class="table table-hover" style="font-size:larger">
+              <tbody>
+                <tr ng-repeat="s in vmTests.f" ng-click="vmTests.trigger = s._id - 1" 
```

Comment:
this is the  weakest part of the trigger approach.
if the _id's of ```f``` arrays are not 1,2,3,4,5... your code will not work.